oarepo_model_builder_multilingual/datatypes/components/multilingual/field.py

```diff
@@ -7,7 +7,9 @@ class MultilingualSchema(ma.Schema):
     class Meta:
         unknown = ma.RAISE
 
-    lang_field = fields.String(data_key="lang-field", required=False)
+    lang_name = fields.String(data_key="lang-name", required=False)
```

Comment:
Please use `attribute` as well and reference as `lang-name` afterwards to minimize confusion (no underscores)